doc/man3/PKCS12_add_safe.pod

```diff
@@ -0,0 +1,63 @@
+=pod
+
+=head1 NAME
+
+PKCS12_add_safe, PKCS12_add_safes - Create and add objects to a PKCS#12 structure
+
+=head1 SYNOPSIS
+
+ #include <openssl/pkcs12.h>
+
+ int PKCS12_add_safe(STACK_OF(PKCS7) **psafes, STACK_OF(PKCS12_SAFEBAG) *bags,
+                    int safe_nid, int iter, const char *pass);
+ PKCS12 *PKCS12_add_safes(STACK_OF(PKCS7) *safes, int p7_nid);
+
+=head1 DESCRIPTION
+
+PKCS12_add_safe() creates a new PKCS7 contentInfo containing the supplied
+B<PKCS12_SAFEBAG>s and adds this to a set of PKCS7 contentInfos. Its type
+depends on the value of B<safe_nid>:
+
+=over 4
+
+=item * If B<safe_nid> is -1, a plain PKCS7 B<data> contentInfo is created.
+
+=item * If B<safe_nid> is a valid PBE algorithm NID, a PKCS7 B<encryptedData>
+contentInfo is created. The algorithm uses B<pass> as the passphrase and B<iter>
+as the iteration count. If B<iter> is zero then a default value for iteration
+count of 2048 is used.
+
+=item * If B<safe_nid> is 0, a PKCS7 B<encryptedData> contentInfo is created using
+a default encryption algorithm, currently B<NID_pbe_WithSHA1And3_Key_TripleDES_CBC>.
+
+=back
+
+PKCS12_add_safes() creates a PKCS12 structure containing the supplied set of
+PKCS7 contentInfos. The B<safes> are enclosed first within a PKCS7 contentInfo
+if type B<p7_nid>. Currently the only supported type is B<NID_pkcs7_data>.
+
+=head1 NOTES
+
+PKCS12_add_safe() makes assumptions regarding the encoding of the given pass
+phrase.
+See L<passphrase-encoding(7)> for more information.
+
+=head1 RETURN VALUES
```

Comment:
These functions were introduced first in OpenSSL 3.0.

Reply:
Documented perhaps ;)